apps/src/templates/export/expo/App.js.ejs

```diff
@@ -100,7 +100,7 @@ export default class App extends React.Component {
 
     return (
       <View onLayout={this.onLayout} style={styles.container}>
-        {!!height && indexUri && <View style={this.webViewContainerStyle()}>
+        {height && indexUri && <View style={this.webViewContainerStyle()}>
```

Comment:
Unrelated change?

Reply:
Yes, somewhat related. I'll put that back in when I restore this work. It was a change needed when I moved to Expo SDK 33, but it could also be there with SDK 31. A simple revert of the whole thing was easier in this case.